json-rpc/tests/integration_test.rs

```diff
@@ -381,6 +382,7 @@ fn create_test_cases() -> Vec<Test> {
                             "base_url_rotation_events_key": format!("0100000000000000{}", address),
                             "compliance_key": "",
                             "compliance_key_rotation_events_key": format!("0000000000000000{}", address),
+                            "diem_id_domains": {"domains": []},
                             "expiration_time": 18446744073709551615_u64,
                             "human_name": "Novi 0",
                             "num_children": 0,
```

Comment:
we should have test to confirm after domain is added, you can read it back by diem_id_domains